CHANGELOG.md

```diff
@@ -22,11 +22,14 @@ Here is a template for new release sections
 ### Added
 - Also components that have no investment costs now have a value (of 0) for COST_INVESTMENT and COST_UPFRONT
 - Display error message when feed-in tariff > electricity price of any  asset in 'energyProvider.csv'. (#497)
+- Added pie plots created using Plotly library to the auto-report (#482) 
+- Added functions to F2_autoreport.py that save the images of plots generated using Plotly to MVS_outputs folder (#499)
```

Comment:
```suggestion
- Added functions to `F2_autoreport.py` that save the images of plots generated using Plotly to `MVS_outputs` folder as `.png` (#499)
```

Is it really the `MVS_outputs` folder, or the output folder determined by the user? I think you use the user-specified folder.
When are the pngs stored - always when the option `-pdf` is chosen? Also when `mvs_report.py` is executed?